### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by PixelSnake** [flutter/flutter#22316]
Flutter crash report; please file at https://github.com/flutter/flutter/issues.

## command

```sh
flutter build bundle --suppress-analytics --target C:\Users\Thomas\Projekte\acm_app\acm_app\lib\main.dart --filesystem-scheme org-dartlang-root --depfile C:\Users\Thomas\Projekte\acm_app\acm_app\build\app\intermediates\flutter\debug/snapshot_blob.bin.d --asset-dir C:\Users\Thomas\Projekte\acm_app\acm_app\build\app\intermediates\flutter\debug/flutter_assets --debug
```
## exception
```
FileSystemException: FileSystemException: Deletion failed, path = 'C:\Users\Thomas\Projekte\acm_app\acm_app\build\app\intermediates\flutter\debug/flutter_assets' (OS Error: Das Verzeichnis ist nicht leer.
, errno = 145)
```

```
#0      _Directory._deleteSync (dart:io/directory_impl.dart:207:7)
#1      FileSystemEntity.deleteSync (dart:io/file_system_entity.dart:466:47)
#2      ForwardingFileSystemEntity.deleteSync (package:file/src/forwarding/forwarding_file_system_entity.dart:72:16)
#3      writeBundle (package:flutter_tools/src/bundle.dart:192:15)
<asynchronous suspension>
#4      assemble (package:flutter_tools/src/bundle.dart:185:9)
<asynchronous suspension>
#5      build (package:flutter_tools/src/bundle.dart:113:9)
<asynchronous suspension>
#6      BuildBundleCommand.runCommand (package:flutter_tools/src/commands/build_bundle.dart:83:11)
<asynchronous suspension>
#7      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:372:18)
<asynchronous suspension>
#8      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:306:33)
<asynchronous suspension>
#9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:142:29)
<asynchronous suspension>
#10     _rootRun (dart:async/zone.dart:1124:13)
#11     _CustomZone.run (dart:async/zone.dart:1021:19)
#12     _runZoned (dart:async/zone.dart:1516:10)
#13     runZoned (dart:async/zone.dart:1463:12)
#14     AppContext.run (package:flutter_tools/src/base/context.dart:141:18)
<asynchronous suspension>
#15     FlutterCommand.run (package:flutter_tools/src/runner/flutter_command.dart:297:20)
#16     CommandRunner.runCommand (package:args/command_runner.dart:194:27)
<asynchronous suspension>
#17     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:347:21)
<asynchronous suspension>
#18     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:142:29)
<asynchronous suspension>
#19     _rootRun (dart:async/zone.dart:1124:13)
#20     _CustomZone.run (dart:async/zone.dart:1021:19)
#21     _runZoned (dart:async/zone.dart:1516:10)
#22     runZoned (dart:async/zone.dart:1463:12)
#23     AppContext.run (package:flutter_tools/src/base/context.dart:141:18)
<asynchronous suspension>
#24     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:303:19)
<asynchronous suspension>
#25     CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:109:29)
#26     new Future.sync (dart:async/future.dart:224:31)
#27     CommandRunner.run (package:args/command_runner.dart:109:11)
#28     FlutterCommandRunner.run (package:flutter_tools/src/runner/flutter_command_runner.dart:211:18)
#29     run.<anonymous closure> (package:flutter_tools/runner.dart:59:20)
<asynchronous suspension>
#30     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:142:29)
<asynchronous suspension>
#31     _rootRun (dart:async/zone.dart:1124:13)
#32     _CustomZone.run (dart:async/zone.dart:1021:19)
#33     _runZoned (dart:async/zone.dart:1516:10)
#34     runZoned (dart:async/zone.dart:1463:12)
#35     AppContext.run (package:flutter_tools/src/base/context.dart:141:18)
<asynchronous suspension>
#36     runInContext (package:flutter_tools/src/context_runner.dart:42:24)
<asynchronous suspension>
#37     run (package:flutter_tools/runner.dart:50:10)
#38     main (package:flutter_tools/executable.dart:52:9)
<asynchronous suspension>
#39     main (file:///E:/Flutter/packages/flutter_tools/bin/flutter_tools.dart:8:3)
#40     _startIsolate.<anonymous closure> (dart:isolate/runtime/libisolate_patch.dart:287:32)
#41     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:171:12)
```

## flutter doctor

```
[✓] Flutter (Channel beta, v0.8.2, on Microsoft Windows [Version 10.0.17134.285], locale de-DE)
    • Flutter version 0.8.2 at E:\Flutter
    • Framework revision 5ab9e70727 (3 weeks ago), 2018-09-07 12:33:05 -0700
    • Engine revision 58a1894a1c
    • Dart version 2.1.0-dev.3.1.flutter-760a9690c2

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at C:\Users\Thomas\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: E:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b02)
    • All Android licenses accepted.

[✓] Android Studio (version 3.1)
    • Android Studio at E:\Program Files\Android\Android Studio
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b02)

[✓] IntelliJ IDEA Community Edition (version 2018.1)
    • IntelliJ at D:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2018.1.4
    • Flutter plugin version 24.2.2
    • Dart plugin version 181.4892.1

[✓] VS Code, 64-bit edition (version 1.23.1)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 2.14.0

[✓] Connected devices (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android P (API 27) (emulator)

• No issues found!
```

**Comment by zoechi** [comment]
> flutter\debug/flutter_assets

Is this a valid path in Windows with mixed forward/backward slashes?

**Comment by PixelSnake** [comment]
Should be, flutter has been started by the flutter plugin for VS Code and launching the app worked prior to this also using VS Code

**Comment by PixelSnake** [comment]
Tried it, gives me the same error:

```
Deleting 'build\'.
FileSystemException: Deletion failed, path = 'build' (OS Error: Das Verzeichnis ist nicht leer.
, errno = 145)
```

FYI, "Das Verzeichnis ist nicht leer" means "The directory is not empty"

**Comment by PixelSnake** [comment]
I assume that too but I only have that problem with the flutter build directory, nothing else. Maybe something goes wrong when creating it?

**Comment by zoechi** [comment]
Hard to tell and I'm not aware of others reporting similar issues.

**Comment by PixelSnake** [comment]
No but I think I figured it out. I got a node backend running in the same directory as the flutter app and I only encounter this error when the backend is running. The error has been caused by nodemon accidentially whatching the build directory, I guess this makes it unable to delete.

I am sorry I wasted your time with an issue that was not related to flutter at all. For the sake of helping others however I would suggest to not delete this issue, maybe someone experiences the same problem in the future.
